lib/PeerRPCClient.js

```diff
@@ -127,6 +127,40 @@ class PeerRPCClient extends PeerRPC {
     )
   }
 
+  mapAll (key, payload, opts = {}, cb) {
+    this.link.lookup(
+      key, {},
+      (err, dests) => {
+        if (err) {
+          cb(err)
+          return
+        }
+
+        if (!dests.length) {
+          return cb(null, [])
+        }
+
+        const limit = opts.limit || 0
+        if (limit) {
+          dests = _.slice(_.shuffle(dests), 0, limit)
+        }
+
+        const excludeDests = opts.excludeDests || false
+        let newDests = dests
+        if (excludeDests) {
+          newDests = dests.filter(dest => !excludeDests.includes(dest))
+        }
+
+        async.map(newDests, (dest, next) => {
+          this.transport(dest, this.getTransportOpts(opts))
+            .request(key, payload, this.getRequestOpts(opts), (err, data) => {
+              if (err) { next(null, err) } else { next(err, data) }
```

Comment:
@face use multiple lines pls

Reply:
@prdn yea, I do use multiple lines, then `eslint standard --fix` puts them on one.   i'll figure out how to stop lint from combing lines into one.